lib/train/version.rb

```diff
@@ -3,5 +3,5 @@
 # Author:: Dominik Richter (<dominik.richter@gmail.com>)
 
 module Train
-  VERSION = '0.15.0'.freeze
+  VERSION = '0.16.0'.freeze
```

Comment:
please lets not make a version bump within a PR. In train, we do a version bump, whenever we release.